spacy_llm/tasks/lemma.py

```diff
@@ -0,0 +1,119 @@
+from typing import Any, Callable, Dict, Iterable, List, Optional, Type
+
+import jinja2
+from spacy.language import Language
+from spacy.scorer import Scorer
+from spacy.tokens import Doc
+from spacy.training import Example
+
+from ..registry import registry
+from ..ty import ExamplesConfigType
+from .templates import read_template
+from .util import SerializableTask
+from .util.examples import LemmaExample
+from .util.serialization import ExampleType
+
+_DEFAULT_LEMMA_TEMPLATE_V1 = read_template("lemma")
+
+
+@registry.llm_tasks("spacy.LEMMA.v1")
```

Comment:
Let's use `Lemma`?

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/164/commits/01790a0148f0a87c5594e9735f97de7f6b1620d6.